db/version_edit_handler.cc

```diff
@@ -824,6 +829,12 @@ void VersionEditHandlerPointInTime::CheckIterationResult(
 
         version_set_->AppendVersion(cfd, v_iter->second);
         versions_.erase(v_iter);
+        // Let's clear found_files, since any files in that are part of the
+        // installed Version. Any files that got obsoleted would have already
+        // been moved to files_to_delete_
```

Comment:
Should `files_to_delete_` be cleared out on each catch up attempt too somewhere?